tests/chainerx_tests/unit_tests/routines_tests/test_manipulation.py

```diff
@@ -617,6 +617,43 @@ def forward_xp(self, inputs, xp):
         return tuple(b)
 
 
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize_pytest('shape,indices_or_sections', [
+    ((2, 4, 6), []),
+    ((2, 4, 6), [2, 4]),
+    ((2, 4, 6), [2, -3]),
+    ((2, 4, 6), [2, 8]),
+    ((2, 4, 6), [4, 2]),
+    ((2, 4, 6), [1, 3]),
+    ((2, 4, 6), numpy.array([1, 2])),  # indices with 1-d numpy array
+    ((2, 4, 6), numpy.array([2])),  # indices with (1,)-shape numpy array
+    ((2, 4, 6), numpy.array(2)),  # sections numpy scalar
+    ((2, 4, 6, 8), numpy.array(2.0)),  # sections with numpy scalar, float
+    ((2, 4, 6, 8), 2.0),  # float type sections, without fraction
+    # indices with empty numpy indices
+    ((2, 4, 6, 8), numpy.array([], numpy.int32)),
+    ((2, 4, 6, 8), numpy.array([], numpy.float64)),
+])
+class TestDSplit(op_utils.NumpyOpTest):
+
+    def setup(self):
+        indices_or_sections = self.indices_or_sections
+        if (isinstance(indices_or_sections, list) and
+                sorted(indices_or_sections) != indices_or_sections):
```

Comment:
Could you comment why do you skip the backward test in this case?
(Could you copy the comment in `TestSplit.setup`?)

Reply:
How about this?

Reply:
i am so sorry. i forgot